JustSaying.IntegrationTests/Fluent/AwsTools/WhenCreatingErrorQueue.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Threading.Tasks;
+using JustSaying.AwsTools;
+using JustSaying.AwsTools.QueueCreation;
+using Microsoft.Extensions.Logging;
+using Shouldly;
+using Xunit.Abstractions;
+
+namespace JustSaying.IntegrationTests.Fluent.AwsTools
+{
+    public class WhenCreatingErrorQueue : IntegrationTestBase
+    {
+        public WhenCreatingErrorQueue(ITestOutputHelper outputHelper)
+            : base(outputHelper)
+        {
+        }
+
+        [AwsFact]
+        public async Task Then_The_Retention_Period_Stays_At_Maximum()
```

Comment:
This test name is a bit misleading as it looks like we're asserting on the retention change (14 days to 100 seconds)

Reply:
39b923565c1d6e41d96b86e04b93150026669179